tools/deploy_postgres.py

```diff
@@ -1,54 +1,66 @@
-import os
+import logging
 import utils
 import deployment_options
+import pvc_size_utils
+
+
+log = logging.getLogger(__name__)
```

Comment:
and where log is going? sure it will be seen?

Reply:
ok i see logging.basicConfig(level=logging.INFO)

Reply:
it gives very bad format to say the truth. If u add logger maybe do it with convenient formatting?